Documentation/articles/getting_started/4_adding_content.md

```diff
@@ -2,29 +2,29 @@
+> For help with creating a project, please look at the [Creating a New Project](0_getting_started.md) section of the Getting Started guide.
 
 ## MonoGame Content Builder Tool (MGCB Editor)
 
-If you have not already, you will need to install the MGCB-Editor for managing your content, for details on getting access to the tool, see the [installation instructions here](~/articles/tools/mgcb_editor.md).
+If you haven't already, you'll need to install the MGCB-Editor for managing your content. For details on getting access to this tool, see the [installation instructions here](~/articles/tools/mgcb_editor.md).
 
-> This is optional, as you can edit the .mgcb files manually if you wish, however, it is recommended to use the tool for the extra features it gives.
+> This is technically optional, since you can edit the .mgcb files manually if you wish, but the editor is highly recommended for ease of use.
 
 ## Adding content
 
-First, you are going to need some content for your game. For this tutorial use the following image of a ball:
+First, you'll need some content for your game. For this tutorial, use the following image of a ball:
 
 ![Open Content](~/images/getting_started/ball.png)
 
 Do **right-click > Save Image As** and save it somewhere locally with the name “ball.png”.
 
-Now open up your game project and look at the Solution explorer window. Expand the **Content** folder and open up **Content.mgcb** file by double-clicking on it.
+Now open up your game project and look at the Solution Explorer window. Expand the **Content** folder and open up **Content.mgcb** file by double-clicking on it.
 
 ![Open Content](~/images/getting_started/3_open_content.png)
 
-You should now see the MGCB Editor window open up. If it does not open up (you see a text file open), then you can right-click on **Content.mgcb** and select **Open With**, then select **mgcb-editor-wpf** in the list, click **Set as Default** and then click **OK**.
+You should now see the MGCB Editor window open up. If a text file opens instead, then right-click on **Content.mgcb** and select **Open With**, then select **mgcb-editor-wpf** in the list, click **Set as Default** and then click **OK**, then try again.
 
-> If you do not see the **mgcb-editor-wpf** option when you right-click and select **Open With**, then please review the [Tools documentation](~/articles/tools/tools.md) for installing the MGCB Editor tool for your operating system.
+> If you do not see the **mgcb-editor-wpf** option when you right-click and select **Open With**, then please review the [Tools documentation](~/articles/tools/tools.md) for installing the MGCB Editor tool for your operating system. If you do see the option, but setting it as default still results in a text file being opened when **Content.mgcb** is clicked, try right-clicking it instead and selecting **Open Containing Folder**. This will open File Explorer, and from there you should be able to double click the MGCB file to open up the editor.
```

Comment:
You don't need to open file explorer, you can use open with, select mgcb-editor-wpf, and than click open. There is a bug in VS that doesn't want to properly set the default editor.